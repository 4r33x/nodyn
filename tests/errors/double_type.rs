
use nodyn::nodyn;

nodyn! {
    pub enum Value {
        FirstName(String),
        LastName(String),
    }
}

fn main() {}
