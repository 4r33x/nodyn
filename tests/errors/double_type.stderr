error: Duplicate variant type detected
 --> tests/errors/double_type.rs:7:18
  |
7 |         LastName(String),
  |                  ^^^^^^
